rbc/heavydb/text_encoding_none.py

```diff
@@ -198,12 +182,58 @@ def heavydb_text_encoding_none_constructor_memcpy(context, builder, sig, args):
     return fa._getpointer()
 
 
+@extending.lower_builtin(TextEncodingNone, nb_types.UnicodeType)
+def text_encoding_none_unicode_ctor(context, builder, sig, args):
+    # Note to the future:
+    # You shall be tempted to use "context.make_helper",
```

Comment:
You said this but used `context.make_helper` in the very first following line of code 😅. I get why `extract_value` is a bad idea, but why the other two? It's not very clear what they do on first glance.

Reply:
Ah yeah, it's pretty safe to assume any code that handles/creates structs directly in LLVM is intended for stack-allocated struct objects and not pointers.